apiserver/facades/client/modelmanager/modelmanager.go

```diff
@@ -1100,7 +1100,12 @@ func (m *ModelManagerAPI) ModelDefaults() (params.ModelDefaultsResult, error) {
 		return result, common.ErrPerm
 	}
 
-	values, err := m.state.ModelConfigDefaultValues()
+	model, err := m.state.Model()
```

Comment:
we already do this in the constructor.
model should be an attribute on the struct so we don't need to do this every time